### ISSUE TITLE
Issue with upload of Interactive Broker trades

### ISSUE THREAD

**Issue by greybox1** [Eleven-Trading/TradeNote#53]
Hi there great script but I am Error in upload an InteractiveBroker file with specified export fields.. getting error (TypeError: Cannot read properties of undefined (reading 'split'))

Looked at addTrades.js and it seems to be using another naming convention for the field names

Would it be possible to include an InteractiveBroker template file?

Thanks!

**Comment by 7aklhz** [comment]
Can you confirm that you followed the instructions here : brokers/README.md ?

If so, the easiest would be to connect via the Discord channel and I will try to help you there.

**Comment by greybox1** [comment]
Thank you for getting back to me so quickly yes I did use your instructions

created these fields

Account ID
Trade Date
Settle Date
Currency
Underlying Symbol
Symbol
Buy/Sell
Quantity
Price
Order Time
Commission
Broker Execution Commission
Broker Clearing Commission
Third-Party Execution Commission
Thirs-Party Clearing Commission <--- spelling mistake
Other Commissions
Proceeds
Net Cash
Trade ID
Order ID

Suggestion
------------
Would have been nice to be able to select all fields took about 5 minutes to choose specific fields. I tried originally export ordering of fields and then re-ordered fields to match instructions.

I have attached example upload.
[TradeDate-IBK.csv](https://github.com/Eleven-Trading/TradeNote/files/11995417/TradeDate-IBK.csv)
